Tasks/plane.js

```diff
@@ -1,17 +1,15 @@
 // Make nested array plane
 
-const_plane = function(arr, res=[]) {
-  j = 0; for(let i = 0, length = arr.length; i < length; i++){
-    value = arr[i];
-    j = i
-    if (Array.isArray(value, typeof value) && [i, length]) {
-    res.push(...const_plane(value));
-    arr[i] = res[i - 1]
-    }else{
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
+const_plane = array => {
+  const res = [];
+  for(const element of array) {
```

Comment:
Make it more readable:
```suggestion
  for (const element of array) {
```